src/utils/global/ambassadorCard/AmbassadorCard.tsx

```diff
@@ -19,6 +23,29 @@ export default function AmbassadorCard(props: AmbassadorCardProps) {
   const images = getAmbassadorImages(ambassadorKey)
   const mod = window?.Twitch?.ext?.viewer?.role === 'broadcaster' || window?.Twitch?.ext?.viewer?.role === 'moderator'
 
+  const [iucnTooltipShown, setIucnTooltipShown] = useState(false);
+
+  const iucnText = "An objective assessment systems for classifying the status of plants, animals, and other organisms threatened with extinction."
+  const iucnInfoIcon = (
+    <button
+      onMouseEnter={() => setIucnTooltipShown(true)}
+      onMouseLeave={() => setIucnTooltipShown(false)}
+      className={styles.iucnInfoIcon}
+    >
+    {/* svg sourced from https://icons.getbootstrap.com/icons/info-circle-fill/. The Tailwind-recommended https://heroicons.com/ wasn't friendly for resizing */}
+    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="gray" viewBox="0 0 16 16">
+      <path d="M8 16A8 8 0 1 0 8 0a8 8 0 0 0 0 16zm.93-9.412-1 4.705c-.07.34.029.533.304.533.194 0 .487-.07.686-.246l-.088.416c-.287.346-.92.598-1.465.598-.703 0-1.002-.422-.808-1.319l.738-3.468c.064-.293.006-.399-.287-.47l-.451-.081.082-.381 2.29-.287zM8 5.5a1 1 0 1 1 0-2 1 1 0 0 1 0 2z"/>
+    </svg>
+    <Tooltip
+      text={iucnText}
+      isShown={iucnTooltipShown}
+      offsetLeft={11}
+      offsetTop={16.75}
+      width={18}
```

Comment:
These values seem very, very specific -- is there not a better way to position this that's just based on the parent etc.?

Reply:
i agree I really tweaked it by the decimal there 😅

i'll think of something. The reason I input those values particularly is since we'd be sharing the Tooltip functionality with the overlay buttons, it's difficult to set position based on percentages because the overlay buttons's 100% evaluates to less than the ambassador's 100%.

thanks a ton for your feedback! I'll get to the changes and implementing Tooltip as a wrapper